webxr/glwindow/mod.rs

```diff
@@ -161,15 +161,14 @@ impl Device for GlWindowDevice {
         self.events.upgrade(callback)
     }
 
-    fn connected(&mut self) -> bool {
-        self.connected
+    fn quit(&mut self) {
+        self.events.callback(Event::SessionEnd);
     }
 
-    fn quit(&mut self) {
-        if !self.connected {
-            self.connected = false;
-            self.events.callback(Event::SessionEnd);
-        }
+    fn set_quitter(&mut self, _: Quitter) {
+        // Glwindow currently doesn't have any way to end its own session
+        // XXXManishearth add something for this that listens for the window
```

Comment:
Yeah, this is annoying. Glutin only has one event loop, which makes it difficult to support examples like this, where events on the servo window should go to servo, and events on the glwindow window should go to it.